packages/squiggle-lang/src/rescript/Utility/E copy.resxxx

```diff
@@ -0,0 +1,916 @@
+/*
```

Comment:
I take it this should be removed?

Reply:
Yes. Thanks for removing forgotten duplicate...